packages/emitter-framework/src/csharp/components/json-converter/json-converter-resolver.tsx

```diff
@@ -0,0 +1,117 @@
+import { useTsp } from "#core/index.js";
+import { createContext, namekey, useContext, type Children, type Namekey } from "@alloy-js/core";
+import {
+  getTypeName,
+  type DurationKnownEncoding,
+  type EncodeData,
+  type Type,
+} from "@typespec/compiler";
+import { getNullableUnionInnerType } from "../utils/nullable-util.js";
+import { TimeSpanIso8601JsonConverter, TimeSpanSecondsJsonConverter } from "./json-converter.jsx";
+
+interface JsonConverterInfo {
+  nameKey: Namekey;
```

Comment:
From the POV of the user of this it should just be a `refkey`
```suggestion
  /** Reference to the json converter class */
  refkey: Refkey;
```

Reply:
Are we on latest Alloy here? If so refkeyable is the way now!

Reply:
@timotheeguerin , @bterlson , any suggestions? thanks.

Reply:
I see, i missed it was the user responsability to render those converters, `namekey` maybe then is fine(casing needs to be changed though)